src/renderer/components/+workloads-replicasets/replicaset-scale-dialog.tsx

```diff
@@ -30,9 +30,10 @@ import { Icon } from "../icon";
 import { Slider } from "../slider";
 import { Notifications } from "../notifications";
 import { cssNames } from "../../utils";
-import { ReplicaSet, replicaSetApi } from "../../api/endpoints/replica-set.api";
+import { ReplicaSet, ReplicaSetApi, replicaSetApi } from "../../../common/k8s-api/endpoints/replica-set.api";
 
 interface Props extends Partial<DialogProps> {
+  replicaSetApi: ReplicaSetApi
```

Comment:
Easier to test if we don't use global stores. And with this PR those global stores are not automatically initialized so had to make these changes.